### PR TITLE
Added western theme buttons to animated

### PR DESCRIPTION
**Pull request by laurachristensen1214**
I added a Button Idea earlier today for leather-looking buttons. I named them "Western themed" and added them to the existing animated buttons.

Issue:

### CODE REVIEW

File: src/components/animated/_western.less [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.choco-btn:hover {
+    background: darken(@choco, 1.5%);
+    color: lighten(@choco, 18%);
+    text-shadow: -1px -1px 0 darken(@choco, 9.5%);
+}
+
+.charcoal-btn {
+    background: lighten(@charcoal, 3%);
```
Comment (shahednasser) [inline src/components/animated/_western.less:41]: You are using `@charcoal` but the variable is not defined

File: assets/js/buttons-examples.js [hunk 2]
```diff
@@ -10,8 +10,13 @@ $(document).ready(function () {
     "yellow-btn",
     "orange-btn",
     "purple-btn",
+    "denim-btn",
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:13]: Why did you add the buttons here? These buttons are just different colors to be used for all button types. Please add them in the `buttons` variable instead under "Special"

File: src/components/animated/_western.less [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.denim-btn {
+    background: lighten(@denim, 3%);
+    text-shadow: -3px -2px 0 darken(@denim, 9.5%);
+    box-shadow: 0px 2px 0 darken(@denim, 5%), 2px 4px 6px darken(@denim, 2%);
+    color: lighten(@denim, 35%);
+}
+
+.denim-btn:hover {
```
Comment (shahednasser) [inline src/components/animated/_western.less:21]: nest this inside `.denim-btn`

File: src/components/animated/_western.less [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.choco-btn {
+    background: lighten(@choco, 3%);
+    text-shadow: -2px -2px 0 darken(@choco, 9.5%);
+    box-shadow: 0px 2px 0 darken(@choco, 5%), 2px 4px 6px darken(@choco, 2%);
+    color: lighten(@choco, 55%);
+}
+
+.choco-btn:hover {
```
Comment (shahednasser) [inline src/components/animated/_western.less:34]: nest this inside `.choco-btn`

File: src/components/animated/_western.less [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.charcoal-btn {
+    background: lighten(@charcoal, 3%);
+    text-shadow: -2px -2px 0 darken(@charcoal, 9.5%);
+    box-shadow: 0px 2px 0 darken(@charcoal, 5%), 2px 4px 6px darken(@charcoal, 2%);
+    color: lighten(@charcoal, 55%);
+}
+
+.charcoal-btn:hover {
```
Comment (shahednasser) [inline src/components/animated/_western.less:47]: nest this inside `.charcoal-btn`

File: src/components/animated/_western.less [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.sand-btn {
+    background: lighten(@sand, 3%);
+    text-shadow: -2px -2px 0 darken(@sand, 9.5%);
+    box-shadow: 0px 2px 0 darken(@sand, 5%), 2px 4px 6px darken(@sand, 2%);
+    color: lighten(@sand, 55%);
+}
+
+.sand-btn:hover {
```
Comment (shahednasser) [inline src/components/animated/_western.less:60]: nest this inside `.sand-btn`

**Review by shahednasser — CHANGES_REQUESTED** [review]
It seems you have created 5 button types? I'm honestly not sure as I can't see the buttons yet. Please make the fixes below to see whether we need the buttons or not and we can proceed afterwards.

**Comment by laurachristensen1214** [comment]
Hi, @shahednasser I am sorry for the delay. This is something I am doing when I have free time in my work schedule, so I needed to put it on the back burner for a couple weeks. I have made the revisions you suggested and would really like to submit the updated pull request if possible?

**Comment by shahednasser** [comment]
Sure @laurachristensen1214 please submit a new PR with your work.

**Pull request closed without merging**